src/styles/components/Code/_base.scss

```diff
@@ -11,15 +11,20 @@
   }
 
   &--line-numbers {
+    $line-sidebar-width: 45px;
+
+    padding-left: $line-sidebar-width;
+
     textarea {
       // Need to use margin here since CodeEditor supports padding: https://github.com/stoplightio/ui-kit/blob/fd1334753a22d527b803b8554bc51761c348d319/src/CodeEditor/index.tsx#L13
       margin-left: 35px !important;
     }
 
     .line-number {
-      padding-left: 45px;
+      margin-left: -$line-sidebar-width;
+      padding-left: $line-sidebar-width;
       position: relative;
-      width: calc(100% - 45px);
+      width: calc(100% - #{line-sidebar-width});
```

Comment:
This is a CSS syntax error. Resolves to `calc(100% - line-sidebar-width)`. You meant to do
```css
width: calc(100% - #{$line-sidebar-width});
```

Reply:
But. As you see it's not needed at all. The rows already expand as much as they want to, there is no point in forcing a 100% width, am I missing something?

Reply:
In case you did need it, it would be `100% + #{$line-sidebar-width}` anyway. Padding doesn't count because of `box-sizing: border-box`, but you need to make up for the negative margin.

Reply:
As discussed, I think it might have been needed by some browser, but we should be good now I suppose.